plugins/gossip/parameters.go

```diff
@@ -13,10 +13,15 @@ const (
 	CfgGossipAgeThreshold = "gossip.ageThreshold"
 	// CfgGossipTipsBroadcastInterval the interval in which the oldest known tip is re-broadcast.
 	CfgGossipTipsBroadcastInterval = "gossip.tipsBroadcaster.interval"
+	// CfgGossipDisableAutopeering if set to true the autopeering layer won't manage neighbors in the gossip layer.
```

Comment:
All these and the "old" parameters can (and maybe should) be simplified using the new framework like here: https://github.com/iotaledger/goshimmer/blob/3f56dd4a2ccf8f086e188f07c6f23ecb741824be/plugins/messagelayer/parameters.go#L10

Reply:
Okay I will look into this

Reply:
Sorry, I don't have time to do it in the scope of this PR. Let's work on this later.